### DISCUSSION TITLE
Implement sliders with accuracy as a separate hitobject type.

### DISCUSSION THREAD

**Discussion by Xinnoh** [ppy/osu#17342] [Ideas]
A compromise that might satisfy both sides of the debate on how accuracy on slider heads should be implemented.
Old sliders and acc sliders could both be implemented in the game, but considered as different object types in the editor. In new maps, acc sliders would used as the default, with old sliders used only as needed (eg weird rhythm). This would settle all issues mappers have with the system.
This would also make it possible to have old maps that rely on the old slider mechanic to use the new score system, only replacing sliders as needed for when the old mechanics are used.

To distinguish them in game, some kind of visual distinction could be added such as thicker slider borders or additional textures on the hitcircle.
RC would be updated with something like "Use acc sliders unless there is a strong reason to use old sliders"

There is precedent for different object types having different timing windows. In maimai, normal buttons have normal timing windows, but touchscreen based taps have much wider timing windows, allowing for the chart makers to use it for complex rhythm as needed.

**Comment by bdach** [comment]
have you tried the classic mod? i think it should be doing what you want.

**Comment by peppy** [comment]
As I said in discord chat, if this setting is exposed to the mapper it will be on a per-beatmap level, and tied to classic mod. Adding a new hitobject type for this purpose is unnecessary complexity IMHO.

**Reply by Xinnoh** [reply]
Ah
Correct me if I'm wrong, but I thought mappers were not going to be allowed to enforce classic mod on future beatmaps (if need be).
If we can then there should be no issue with the state of things as they are.
